budget/budget.w

```diff
@@ -0,0 +1,17 @@
+bring util;
+
+bring "./budget-tfaws.w" as tfaws;
+bring "./budget-shared.w" as shared;
+
+pub class Budget {
+  platform: shared.IBudget;
+
+  new(props: shared.BudgetProps) {
+    if util.env("WING_TARGET") == "tf-aws" {
+      this.platform = new tfaws.BudgetTfAws(props);
+    }
+    else {
+      throw "unknown platform";
```

Comment:
We should at least add some basic support for `sim`. In this case maybe it's just a no-op but it should not throw.

Reply:
:open_mouth: my mistake